python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/dhcp_servers.schema.yml

```diff
@@ -19,6 +19,48 @@ keys:
           description: VRF in which to configure the DHCP server, use `default` to indicate default VRF.
           convert_types:
           - int
+        lease_time_ipv4:
+          type: dict
+          keys:
+            days:
+              type: int
+              convert_types:
+              - str
+              min: 0
+              max: 2000
+            hours:
+              type: int
+              convert_types:
+              - str
+              min: 0
+              max: 23
+            minutes:
+              type: int
+              convert_types:
+              - str
+              min: 0
+              max: 59
+        lease_time_ipv6:
+          type: dict
+          keys:
+            days:
+              type: int
+              convert_types:
+              - str
+              min: 0
+              max: 2000
+            hours:
+              type: int
+              convert_types:
```

Comment:
It would be easy if we keep default 0 for hours and minutes.

Reply:
As discussed, keeping it as required.